ocs_ci/templates/workloads/jenkins/jenkins_template

```diff
@@ -0,0 +1,246 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+labels:
+  app: jenkins-persistent-ocs
+  template: jenkins-persistent-ocs-template
+message: A Jenkins service has been created in your project.  Log into Jenkins with
+  your OpenShift account.  The tutorial at https://github.com/openshift/origin/blob/master/examples/jenkins/README.md
+  contains more information about using this template.
+metadata:
+  annotations:
+    description: |-
+      Jenkins service, with persistent storage.
+
+      NOTE: You must have persistent volumes available in your cluster to use this template.
+    iconClass: icon-jenkins
+    openshift.io/display-name: Jenkins
+    openshift.io/documentation-url: https://docs.okd.io/latest/using_images/other_images/jenkins.html
+    openshift.io/long-description: This template deploys a Jenkins server capable
+      of managing OpenShift Pipeline builds and supporting OpenShift-based oauth login.
+    openshift.io/provider-display-name: Red Hat, Inc.
+    openshift.io/support-url: https://access.redhat.com
+    samples.operator.openshift.io/version: 4.4.0-0.nightly-2020-06-08-083627
```

Comment:
Is this OCP version?  
Without hardcoding is it possible to take the latest version here